README.md

```diff
@@ -302,6 +302,7 @@ The 🛠 emoji indicates that a rule is automatically fixable by the `--fix` com
 | C414 | UnnecessaryDoubleCastOrProcess | Unnecessary <list/reversed/set/sorted/tuple> call within <list/set/sorted/tuple>(). |  |  |
 | C415 | UnnecessarySubscriptReversal | Unnecessary subscript reversal of iterable within <reversed/set/sorted>() |  |  |
 | C416 | UnnecessaryComprehension |  Unnecessary <list/set> comprehension - rewrite using <list/set>() |  |  |
+| C417 | UnnecessaryMap | Unnecessary map usage - rewrite using a <list/set/dict> comprehension |  |  |
```

Comment:
@harupy - Do you want to do the honor of removing `(15/16)` from the README?

Reply:
Removed!